test/extensions/filters/network/common/fuzz/BUILD

```diff
@@ -85,15 +99,28 @@ envoy_cc_fuzz_test(
     srcs = ["network_readfilter_fuzz_test.cc"],
     corpus = "network_readfilter_corpus",
     dictionaries = ["network_readfilter_fuzz_test.dict"],
-    # All Envoy network filters must be linked to the test in order for the fuzzer to pick
-    # these up via the NamedNetworkFilterConfigFactory.
     deps = [
         ":uber_readfilter_lib",
         ":vig_anymap_ext_lib",
         "//source/common/config:utility_lib",
         "//test/config:utility_lib",
         "//test/test_common:test_runtime_lib",
-    ] + envoy_all_network_filters(),
+    ] + fuzz_filters_targets(_readfilter_fuzz_filters),
```

Comment:
Instead of calling `fuzz_filters_targets`, why not have a function in source/extensions/all_extensions.bzl that receives the list of extensions, and returns the `_selected_extension_target()` of these extensions?

Reply:
Is this what you had in mind: [c0a9683](https://github.com/envoyproxy/envoy/pull/27373/commits/c0a96834a593802ca57815f3e1ae680543984069l)?